demo/src/client/faro/initialize.ts

```diff
@@ -18,6 +18,7 @@ export function initializeFaro(): Faro {
     instrumentations: [
       ...getWebInstrumentations({
         captureConsole: true,
+        captureWebVitalsAttribution: true,
```

Comment:
Is there any problem with creating this option to facilitate the adoption of the instrumentation?

Reply:
Let's do this a bit differently. 

For the user web-vital tracking should be transparent.

Configuring the web-vitals instrumentation should happen via a setting in the Faro init code instead.

For example:

```ts
const faro = coreInit({  
    instrumentations: [],
    trackWebVitalAttribution: true,
  });
```

Under the hood we can keep them as separate instruments. 
For example we can introduce a main init() func for the web vitals instrumentation which decided which instrument to load/initialize by the above setting.

Reply:
I made the change here: https://github.com/grafana/faro-web-sdk/pull/595/commits/a25d91c68b093b993cd30d2183ac264eb592ca4f

I'm not sure if I understand the idea well so I made everything in a single commit to make it easy to revert if necessary

for some reason the `faro.config` is `undefined` in the context of `getWebInstrumentations`. So I needed to put everything together on the same instrumentation.

I was unable to do a dynamic import because the package is not esm, I believe the same problem that [I mentioned here](https://github.com/grafana/faro-web-sdk/pull/595#discussion_r1605892211) continues to happen

This means I have both `web-vitals` and `web-vitals/attribution` in the bundle, right? even if the user is just using web vitals basic

```ts
    if (faro.config.trackWebVitalAttribution) {
      return new WebVitalsWithAttribution(this.api.pushMeasurement)
    }
    return new WebVitalsBasic(this.api.pushMeasurement)
```

Reply:
This is perfectly that way, thanks.
Yes you have both instruments in the bundle. But it would be the same if we use the `getWebVitals()` function.